### DISCUSSION TITLE
BonBon 1.x

### DISCUSSION THREAD

**Discussion by danielfebrero** [BonBon-exchange/bonbon-web-browser#479] [Ideas]
# BonBon 1.x

## Abstract

BonBon Browser 0.30.6 has been available for 2 years. No moderate issues have been reported. Thousands of unique users have been convinced of its utility.

BonBon is entering the magic list of a growing number of people. The list of softwares you install with no condition when you start from scratch.

## Objectives

- reward early users with new features, improved experience and security
- implement last miles to pass the door of exigent controllers such as chatgpt.com and web.whatsapp.com
- increase control users have on the visual experience
- increase control users have on the technical possibilities
- implement missing connectors to leverage the best of chrome extension compatibility and manifest 2
- provide a way to reference and install easily chrome extension manifest 2
- add one or two features for wahou effect

## Epics

- magic chat
- settings for visual
- manifest 2 (missing connectors and library)
- issues backlog

**Comment by danielfebrero** [comment]
@detherminal @dcallus @PerpetualOwl @0xCUB3 @Nuklusone

**Reply by 0xCUB3** [reply]
Awesome!

**Comment by danielfebrero** [comment]
have a look at @ in the left bar... that's the magic chat!
from there, you gonna enter ... a bar! yeah, like a party bar. you click on it, and wow, there are people around.
they have ideas, values, interests... let's make social! 

i'm so excited :D

<img width="1470" alt="Capture d’écran 2024-08-04 à 17 15 06" src="https://github.com/user-attachments/assets/0f16e15e-5b48-4582-90b9-7ad0db437aee">

**Comment by danielfebrero** [comment]
Created PR (WIP) for opening new webpages or new boards in a separate session (multi login, beginin of incognito, etc.)

- DONE: ctrl + t + s to open a webpage in a separate session

https://github.com/BonBon-exchange/bonbon-browser/pull/510
